packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-selectors.ts

```diff
@@ -36,7 +36,7 @@ export const GeneratedAnswerSelectors: GeneratedAnswerSelector = {
   get: () => cy.get(generatedAnswerComponent),
 
   generatedAnswerCard: () =>
-    GeneratedAnswerSelectors.get().find('[data-cy="generated-answer__card"]'),
+    GeneratedAnswerSelectors.get().find('[data-test="generated-answer__card"]'),
```

Comment:
Here I started moving gradually to `data-test` because there are some selectors that already exist in the Cypress tests and that I wanted to reuse in the new unit tests, so instead of creating more places that use data-cy that we know will be removed soon, I thought about gradually starting to switch to data-test, and reduce the places where we use data-cy.

and of course when delete our cypress setup, we remove every remaining occurrence of data-cy.

To summarize, I wanna avoid having data-cy in our unit tests as this increase the volume of work needed when cleaning out all cypress and data-cy.

Reply:
I updated your pr to `data-testid` for future playwright.